src/sdk/model/vm/guest-tools/guest-tools.ts

```diff
@@ -0,0 +1,60 @@
+import { GuestToolsJson } from './__json__';
+
+/**
+ * VM Guest Tools
+ */
+export class GuestTools {
+  private _json: GuestToolsJson;
+
+  constructor(guestTools: GuestTools);
+  constructor(guestToolsJson: GuestToolsJson);
+  constructor(firstParam: boolean | GuestTools | GuestToolsJson) {
```

Comment:
When would we have a boolean as firstParam there ? I can't think of any cases...